great-idea-website/index.html

```diff
@@ -1,63 +1,125 @@
+      <header>
+        <nav>
+          <a href="#">Services</a> 
+          <a href="#">Product</a>
+          <a href="#">Vision</a> 
+          <a href="#">Features</a> 
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </nav>
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo." />
+      </header>
+
+      <section class="first-section">
+        <div class="first-section-text">
+          <h1>
+            Innovation<br />
+            On <br />
+            Demand
+          </h1>
+          <div class="button"><button>Get Started</button>
+          </div>
+        </div>
+
+        <div class="first-section-text">
+          <img src="img/header-img.png" alt="Image of a code snippet." />
+        </div>
+      </section>
+
+      <section class="second-section">
+        <div class="second-section-text">
+          <h2>Features</h2>
+          <p>
+            Aliquam elementum magna eros, ac posuere elvit tempus et.
+            Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu
+            ornare nisl. Nullam convallis augue justo, at imperdiet metus
+            scelerisque quis.
+          </p>
+        </div>
+
+        <div class="second-section-text">
+          <h2>About</h2>
+          <p>
+            Aliquam elementum magna eros, ac posuere elvit tempus et.
+            Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu
+            ornare nisl. Nullam convallis augue justo, at imperdiet metus
+            scelerisque quis.
+          </p>
+        </div>
+      </section>
+
+      <section>
+        <img
+          class="middle-img"
+          src="img/mid-page-accent.jpg"
+          alt="Image of code snippets across the screen"
+        />
+      </section>
+
+      <section class="third-section">
+        <div class="third-section-text">
+          <h2>Services</h2>
+          <p>
+            Aliquam elementum magna eros, ac posuere elvit tempus et.
+            Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu
+            ornare nisl. Nullam convallis augue justo, at imperdiet metus
+            scelerisque quis.
+          </p>
+        </div>
+
+        <div class="third-section-text">
+          <h2>Product</h2>
+          <p>
+            Aliquam elementum magna eros, ac posuere elvit tempus et.
+            Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu
+            ornare nisl. Nullam convallis augue justo, at imperdiet metus
+            scelerisque quis.
+          </p>
+        </div>
+
+        <div class="third-section-text">
+          <h2>Vision</h2>
+          <p>
+            Aliquam elementum magna eros, ac posuere elvit tempus et.
+            Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu
+            ornare nisl. Nullam convallis augue justo, at imperdiet metus
+            scelerisque quis.
+          </p>
+        </div>
+      </section>
+
+      <address>
```

Comment:
It might make more sense to move the opening address tag to beneath the contact h2 here.